scripts/delayed.js

```diff
@@ -1,7 +1,64 @@
 // eslint-disable-next-line import/no-cycle
-import { sampleRUM } from './lib-franklin.js';
+import { sampleRUM, getMetadata } from './lib-franklin.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
 // add more delayed functionality here
+
+function loadGoogleTagManager() {
+  // Initialize the data layer
+  /* global userdata */
+  /* eslint no-undef: "error" */
+  const locale = getMetadata('locale');
+  const pageCountry = {
+    en: 'INT',
+    de: 'DE',
+  };
+  const conf = {
+    pageArea: 'web',
+    pageCountry: pageCountry[locale],
+    pageEnvironment: 'prod_publish',
+    pageIdentifier: 'main',
+    pageLanguage: locale,
+    pageLocation: 'ALL_General-Article_002',
+    pageName: window.location.href,
+    pagePool: 'smt',
+    pageTags: '',
+    contentHierarchy1: 'news-and-events',
+    productName: '',
+  };
+  if (typeof userdata !== 'undefined' && userdata.accountid) {
```

Comment:
How is `userdata` initialised? Through some external mechanism?

Reply:
@sdmcraft I suggest to capture user data management in a separate issue. wdyt ?